config/routes.rb

```diff
@@ -1,3 +1,5 @@
 Rails.application.routes.draw do
+  get 'mentors/index'
+
   root to: 'pages#index'
```

Comment:
semantics thing, rails styling guide, can we move the new route below root. root should always come first.. it is the root.. 😄